docs/release-files.md

```diff
@@ -0,0 +1,75 @@
+Releasing hsd
+=============
+
+This document contains information about bundling, signing and
+distributing the files.
+
+`hsd` is distributed through several platforms: `github`, `npm`, `brew`.
+
+<!-- markdown-toc -i release-files.md -->
+
+<!-- toc -->
+
+- [Deploying to github (tag)](#deploying-to-github-tag)
+  * [Major, minor and patches](#major-minor-and-patches)
+- [Deploying to npm](#deploying-to-npm)
+  * [Deploying latest version, minor and patches included](#deploying-latest-version-minor-and-patches-included)
+  * [Deploying support versions (previous and life-support)](#deploying-support-versions-previous-and-life-support)
+- [Deploying to homebrew](#deploying-to-homebrew)
+
+<!-- tocstop -->
+
+## Deploying to github (tag)
+
+  This does not need many additional actions as we use github as our primary
+git platform. After each release we need to update INSTALL.md versions. Tags
+MUST to be signed.
+
+### Major, minor and patches
+After you are [ready](./release-process.md) to publish files
+(for example `v4.0.0`)
+  - `git checkout 4.x` - switch to proper branch `4.x`.
+  - `git tag -s v4.0.0` - tag and sign the release.
+  - `git push origin v4.0.0` - publish the tag. (origin is the main repo)
```

Comment:
may also require `git push --tags` as separate command

Reply:
Yeah, in that case author needs to have clean tags list, no custom or private tags. (I don't see why you would have custom/local tags, but you never know)

Reply:
oh shit I didn't think about that. then `git push origin <tagname>` is sufficient.